src/sql/workbench/browser/modelComponents/declarativeTable.component.html

```diff
@@ -1,45 +1,52 @@
 <table role=grid #container *ngIf="columns" class="declarative-table" [style.height]="getHeight()"
 	[style.width]="getWidth()" [attr.aria-label]="ariaLabel">
-	<thead>
-		<ng-container *ngFor="let column of columns; let c = index;">
-			<th class="declarative-table-header" aria-sort="none" [style.width]="getColumnWidth(column)"
-				[attr.aria-label]="column.ariaLabel" [ngStyle]="column.headerCssStyles" role="columnheader">
-				{{column.displayName}}
-				<checkbox *ngIf="isCheckBox(c)" [checked]="isHeaderChecked(c)"
-					(onChange)="onHeaderCheckBoxChanged($event,c)" label=""></checkbox>
-			</th>
-		</ng-container>
+	<thead role="rolegroup">
+		<tr role="row">
+			<ng-container *ngFor="let column of columns; let c = index;">
+				<th class="declarative-table-header" aria-sort="none" [style.width]="getColumnWidth(column)"
+					[ngStyle]="column.headerCssStyles"
+					[attr.aria-label]="(column.ariaLabel)?column.ariaLabel:column.displayName">
```

Comment:
Let's keep logic out of the templates best we can - either update ariaLabel getter to use displayName by default or (my preferred) make a separate function called like `getHeaderAriaLabel` (so that callers can get the actual aria label property set if needed)

(similar to how we get the cell aria label)